docs/source/examples/best-practices.rst

```diff
@@ -44,6 +44,14 @@ We also recommend allocating most, though not all, of the GPU memory space. We d
 
 Additionally, when using `Accelerated Networking`_ , we only need to register a single IPC handle for the whole pool (which is expensive, but only done once) since from the IPC point of viewer there's only a single allocation. As opposed to just using RMM without a pool where each new allocation must be registered with IPC.
 
+Spilling from Device
+~~~~~~~~~~~~~~~~~~~~
+
+Dask CUDA offers several different ways to enable automatic spilling from device memory.
```

Comment:
```suggestion
Dask-CUDA offers several different ways to enable automatic spilling from device memory.
```

Reply:
What is the "grammar" rule on hyphenating software/project names? I faintly remember discussing this with @jacobtomlinson at one point, and I though I left with the sense that the hyphen shouldn't be there. However, there is a good chance I'm remembering incorrectly.

Should we add the hyphen back across all the Dask cuDF documentation? (https://github.com/rapidsai/cudf/pull/16765)

Reply:
I don't know about the grammar rule, but technically this is a name and I wouldn't think any grammar necessarily applies. We've been using Dask-CUDA exclusively so it matches the project/repo name `dask-cuda`, although I saw now that the most obvious place, [README.md](https://github.com/rapidsai/dask-cuda/blob/branch-24.10/README.md#dask-cuda), actually spells `Dask CUDA` but that's the only place.

For the reason stated above I'd prefer to keep `Dask-CUDA`, but if there are strong/compelling reasons to change I'd be open for that. As for `Dask cuDF` I won't be too opinionated, but to me it feels like it makes more sense to match the project name `dask-cudf`, thus with a hyphen.

Reply:
Yeah, sorry. I probably should have left this comment on one of the `Dask cuDF -> Dask-cuDF` suggestions. I don't think there is actually a clear "grammar" rule to point to. It should be fine to just choose a name like "Dask-CUDA", and use it everywhere. It looks slightly funny to my eye, but I have no problem using it.

For `Dask cuDF`, I do have a small preference to leave out the hyphen (it's more pleasing to my eye and is more consistent with Dask DataFrame is written). I think the project name means much, since it's not really possible to use an empty space in that context anyway.

Reply:
In the case of Dask-CUDA to me it feels more natural with the hyphen as well to identify we're talking about the project itself, and not using CUDA with Dask which is perfectly doable too.